terra-scm-1.rockspec

```diff
@@ -0,0 +1,34 @@
+package = "terra"
+version = "scm-1"
+
+source = {
+   url = "git://github.com/zdevito/terra.git",
+   branch = "master",
+}
+
+description = {
+   summary = "A low-level counterpart to Lua.",
+   homepage = "",
+   license = "MIT",
+}
+
+build = {
+   type = "command",
+   build_command = [[
+   make clean
+
+    make LUAJIT_LIB=$(LUA_LIBDIR)/libluajit-5.1.a \
```

Comment:
This is the iffy part. It'd be great if we weren't so dependent on very specific library and directory names. Do you think the Makefile is the right place to add robustness?